rollingpin/args.py

```diff
@@ -243,3 +257,28 @@ def construct_canonical_commandline(config, args):
             arg_list.extend(command)
 
     return " ".join(arg_list)
+
+
+def _get_available_profiles(profile_dir):
+    profiles = []
+    for f in os.listdir(profile_dir):
+        if f.endswith(".ini"):
+            profile_name, _ = f.split(".", 1)
+            profiles.append(profile_name)
+
+    return profiles
+
+def make_profile_parser(profile_dir="/etc/rollingpin.d/",
```

Comment:
Would be nice to define this `/etc/rollingpin.d/` constant once and re-use it throughout the codebase